docs/SOPs/Managed_access/Review_and_export_SOP.md

```diff
@@ -14,13 +14,14 @@ Once the wrangler has verified that the data and metadata files were transferred
 
 ## Submission review and validation
 Data and metadata in a managed access submission are considered sensitive information and must be kept encrypted. Unlike in the case of open access projects, review for a managed access project must be done without ever downloading the metadata spreadsheet - or the data - to a non-encrypted location, like a laptop.
-1. Once the metadata spreadsheet is uploaded to the secure AWS storage area a lambda function will automatically trigger the upload of the spreadsheet to the respective project in the HCA Data Repository Ingest Service.
-2. If the spreadsheet upload fails because HCA Data Repository Ingest Service cannot process the spreadsheet, the wrangler will get notified of the problem. Get in touch with the contributor and suggest to review their spreadsheet over a video call. The contributor should share their screen and walk through the spreadsheet.
-3. If the spreadsheet imports successfully in HCA Data Repository Ingest Service, review the submission for metadata errors and communicate with the contributor on how to fix them. Ask the contributor to upload the corrected spreadsheet to the secure storage area and repeat the activities from step 1 onwards. 
-4. Sync the data files directly from the secure storage area provided to the contributor to the HCA Data Repository Ingest Service AWS staging area \
+1. Once the metadata spreadsheet is uploaded to the secure AWS storage area a lambda function will automatically notify the wrangler.
+2. The wrangler can review and edit the spreadsheet in a secure space through aws WorkSpace - [see here for set-up instructions]([url](https://github.com/ebi-ait/dcp-secure-spreadsheet-upload/wiki))
```

Comment:
you could type only 1. for each of the numbered rows, and markdown converts it to the appropriate number

Reply:
I need to fix the link here!